classes/components/forms/context/PKPEmailSetupForm.php

```diff
@@ -61,17 +62,7 @@ public function __construct($action, $locales, $context)
                 [
                     'key' => 'contextName',
                     'description' => __('emailTemplate.variable.context.contextName'),
-                    'value' => '{$contextName}',
-                ],
-                [
-                    'key' => 'senderName',
-                    'description' => __('emailTemplate.variable.sender.senderName'),
-                    'value' => '{$senderName}',
-                ],
-                [
-                    'key' => 'senderEmail',
-                    'description' => __('emailTemplate.variable.sender.senderEmail'),
-                    'value' => '{$senderEmail}',
+                    'value' => '{$' . ContextEmailVariable::CONTEXT_NAME . '}',
```

Comment:
Instead of manually setting this list of variables, can we generate it from the `ContextEmailVariable::descriptions()`?

Reply:
Done